data/timeline.json

```
@@ -2,121 +2,165 @@
   {
     "type": "internship",
     "title": "Cyber-Security Internship",
-    "institute": "Mezu",
-    "location": {
-      "city": "Lisbon",
-      "country": "Portugal"
-    },
+    "institutes": [
+      {
+        "name": "Mezu",
+        "location": {
+          "city": "Lisbon",
+          "country": "Portugal"
+        }
+      }
+    ],
     "when": 2019,
     "link": "https://www.mezu.com/"
   },
   {
     "type": "volunteer",
     "title": "Slush Volunteer",
-    "institute": "Slush",
-    "location": {
-      "city": "Helsinki",
-      "country": "Finland"
-    },
+    "institutes": [
+      {
+        "name": "Slush",
+        "location": {
+          "city": "Helsinki",
+          "country": "Finland"
+        }
+      }
+    ],
     "when": 2018,
     "link": "https://www.slush.org/"
   },
   {
     "type": "education",
     "title": "Started M.Sc Security & Cloud Computing",
-    "institute": "Aalto University",
-    "location": {
-      "city": "Espoo",
-      "country": "Finland"
-    },
+    "institutes": [
+      {
+        "name": "Aalto University",
+        "location": {
+          "city": "Espoo",
+          "country": "Finland"
+        }
+      }
+    ],
     "when": 2018,
     "link": "http://secclo.aalto.fi/en/"
   },
   {
     "type": "education",
     "title": "Graduated B.Sc Computer Science",
-    "institute": "TU Delft",
-    "location": {
-      "city": "Delft",
-      "country": "Netherlands"
-    },
+    "institutes": [
+      {
+        "name": "TU Delft",
+        "location": {
+          "city": "Delft",
+          "country": "Netherlands"
+        }
+      }
+    ],
     "when": 2018,
-    "link": "/downloads/bsc-computer-science_tu-delft-2018.pdf"
+    "link": "https://www.optiver.com/na/en/our-locations/amsterdam"
   },
   {
     "type": "internship",
     "title": "Data Analytics Internship",
-    "institute": "Optiver Amsterdam",
-    "location": {
-      "city": "Amsterdam",
-      "country": "Netherlands"
-    },
+    "institutes": [
+      {
+        "name": "Optiver Amsterdam",
+        "location": {
+          "city": "Amsterdam",
+          "country": "Netherlands"
+        }
+      }
+    ],
     "when": 2018,
-    "link": "https://www.optiver.com/na/en/our-locations/amsterdam"
+    "link": "/downloads/bsc-computer-science_tu-delft-2018.pdf"
```

This got mixed up with the URL for graduating :sweat_smile: 

```suggestion
    "link": "https://www.optiver.com/na/en/our-locations/amsterdam"
```